Sources/BottomSheet/BottomSheet.swift

```diff
@@ -85,7 +88,11 @@ public struct BottomSheet<Content: View>: View {
             .opacity(grayBackgroundOpacity)
             .edgesIgnoringSafeArea(.all)
             .animation(.interactiveSpring())
-            .onTapGesture { self.isPresented = false }
+            .onTapGesture {
+                self.isPresented = false
+                guard let onDismiss = onDismiss else { return }
```

Comment:
we probably won't need to unwrap it? `onDismiss?()` should be good

Reply:
I fixed it up as you requested. Thank you for your support!